api/python/quilt3/main.py

```diff
@@ -334,6 +342,37 @@ def create_parser():
     )
     verify_p.set_defaults(func=cmd_verify)
 
+    # push
+    shorthelp = "Pushes the new package to the remote registry"
+    push_p = subparsers.add_parser("push", description=shorthelp, help=shorthelp, allow_abbrev=False)
+    push_p.add_argument(
+        "name",
+        help="Name of package, in the USER/PKG format",
+        type=str,
+    )
+    push_p.add_argument(
+        "--dir",
+        help="Directory to add to the new package",
+        type=str,
+        required=True,
+    )
+    push_p.add_argument(
+        "--registry",
+        help="Registry where to create the new package. Defaults to the default remote registry.",
+        type=str,
+    )
+    push_p.add_argument(
+        "--dest",
+        help="Where to copy the objects in the package",
+        type=str,
+    )
```

Comment:
should we make it positional?

Reply:
only if the underlying function supports a positional dest; in which case CLI should be both positional and keyword